contracts/DebondToken.sol

```diff
@@ -255,4 +255,8 @@ abstract contract DebondToken is IDebondToken, ERC20, GovernanceOwnable {
         );
         exchangeAddress = _exchangeAddress;
     }
+
+    function burn(uint amount) external {
```

Comment:
I considered that the issue was the **burn** function was missing in the **DebondToken** contract. In that case, the **burn** function added by @toufic0710 to burn Debond tokens has no issue since it's similar to the **mint** function that was already there. Therefore, this function can be used safely.